src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -360,14 +363,18 @@ private void createConference(
             = JitsiMeetGlobalConfig.getGlobalConfig(
                 FocusBundleActivator.bundleContext);
 
-        String id = generateConferenceId();
-        JitsiMeetConferenceImpl conference
-            = new JitsiMeetConferenceImpl(
+        JitsiMeetConferenceImpl conference;
+        String id;
+        synchronized (conferencesSyncRoot)
+        {
+            id = generateConferenceId();
```

Comment:
this can technically be outside, no? since it grabs it on its own?  probably doesn't make a difference i guess.